frontend/src/models/types/RubricCriterion.ts

```diff
@@ -22,20 +30,20 @@ export const calcMaxPoints = (ratings: Rating[]): number => {
 /**
  * Criterion factory function.
  */
-export default function createCriterion(
+export default function createRubricCriterion(
   description: string = '',
   longDescription: string = '',
   points: number = 0,
-  ratings: Rating[] = [],
+  ratings: RubricRating[] = [],
   id: number | undefined = undefined
```

Comment:
undefined id was causing issues with the handleSubmitRubric logic

locally I changed new rubric objects to default to -1 without an id and then everything worked properly!
<img width="450" alt="image" src="https://github.com/user-attachments/assets/f0ea6c69-07a2-49c2-a5ca-30c444773211">

Reply:
I think once we refactor our types, that'll help clean this up a bit more too

Reply:
Yeah being specific probably would have been helpful 😆 

Prisma was throwing an error because even though it was searching by title, it was still trying to pass in a required id field. I think it's how our types were setup in this branch though but I guess we'll find out if anything blows up soon.